### PR TITLE
Made getfield function in ndarray.py

### LINKED ISSUE #19634: getfield

### PR DESCRIPTION
**Pull request by androozhang**
Close #19634

### CODE REVIEW

**Comment by KevinUli** [comment]
Hi, could you add tests for the frontend method you implemented as shown [here](https://unify.ai/docs/ivy/overview/deep_dive/ivy_frontends_tests.html)? Feel free to request a re-review after.

**Review by KevinUli — COMMENTED** [review]
Hi, could you add tests for the frontend method you implemented as shown [here](https://unify.ai/docs/ivy/overview/deep_dive/ivy_frontends_tests.html)? Feel free to request a re-review after.

**Comment by KevinUli** [comment]
Please also test for `dtype` and `offset` kwarg.

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py [hunk 1]
```diff
@@ -3275,3 +3275,40 @@ def test_numpy_instance_invert__(
...
+):
+    input_dtypes, x = dtype_and_x
+    helpers.test_frontend_method(
+        init_input_dtypes=input_dtypes,
+        init_all_as_kwargs_np={
+            "object": x[0],
+        },
+        method_input_dtypes=input_dtypes,
+        method_all_as_kwargs_np={
+            "value": x[1],
```
Comment (KevinUli) [inline ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py:3307]: The test also seems to be failing due to `value` not being a kwarg accepted by the method.

**Comment by paulaehab** [comment]
Hi @shuoandrew could you please do the changes requested by @KevinUli

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.

**Pull request closed without merging**